packages/frontend/src/hooks/useCrabPosition.ts

```diff
@@ -8,14 +8,14 @@ import { useAtomValue } from 'jotai'
 import { useTokenBalance } from './contracts/useTokenBalance'
 import { addressesAtom } from 'src/state/positions/atoms'
 import { indexAtom } from 'src/state/controller/atoms'
+import useAppCallback from './useAppCallback'
```

Comment:
what is useAppCallback? / how does it differ from normal useCallback?

Reply:
This is a new hook added by @adam5909099  https://github.com/opynfinance/squeeth-monorepo/pull/272 Helps to do the `.toString` of BigNumbers automatically so we don't have to ignore any warnings thereby missing any important dependency is missed.

Reply:
Also can't have tests until we find out how to test PnLs correctly